src/GraphWebhooks/Graph/ChangeNotificationCollectionExtensions.cs

```diff
@@ -34,32 +30,52 @@ public static class ChangeNotificationCollectionExtensions
     /// <param name="issuerPrefix">The prefix for valid issuers (Default: https://login.microsoftonline.com/)</param>
     /// <returns>true if all tokens are valid, false otherwise</returns>
     /// <exception cref="ArgumentNullException"></exception>
-    public static async Task<bool> AreTokensValidV2(
+    public static async Task<bool> AreTokensValid(
         this ChangeNotificationCollection collection,
         IEnumerable<Guid> tenantIds,
         IEnumerable<Guid> appIds,
         string wellKnownUri = "https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration",
-        string issuerPrefix = "https://login.microsoftonline.com/")
+        IEnumerable<string>? issuerFormats = null)
```

Comment:
```suggestion
        IEnumerable<string> issuerFormats = [])
```
I think you can now do that with CSHarp 12 which should simplify the rest of your code

Reply:
Compiler doesn't like it. `Default parameter value for 'i
ssuerFormats' must be a compile-time constant`.

Reply:
ah ok, forget about it then. I yet yet to get used to the subtilities of new C# 12 features. Resolving